lib/stb_image.ex

```diff
@@ -32,18 +36,21 @@ defmodule StbImage do
 
   ## Example
 
-      {:ok, img, shape, type, channels} = StbImage.from_file("/path/to/image")
-      {h, w, c} = shape
+      stb_image_struct = StbImage.from_file("/path/to/image")
+      {h, w, c} = stb_image_struct.shape
+      img = stb_image_struct.data
```

Comment:
```suggestion
      img = StbImage.from_file("/path/to/image")
      {h, w, c} = img.shape
      data = img.data
```

@josevalim do we want typespecs for the struct to document the fields instead?

Reply:
We can also add typespecs later!